.circleci/config.yml

```diff
@@ -1,23 +1,39 @@
 version: 2
 
-defaults: &defaults
-  working_directory: ~/project/ethereumjs-vm
-  docker:
-    - image: circleci/node:8-browsers
-restore_node_modules: &restore_node_modules
-  restore_cache:
-    name: Restore node_modules cache
-    keys:
-      - v1-node-{{ .Branch }}-{{ checksum "package.json" }}
-      - v1-node-{{ checksum "package.json" }}
+aliases: 
+  - &defaults
+    working_directory: ~/project/ethereumjs-vm
+    docker:
+      - image: circleci/node:8-browsers
+
+  - &restore_node_modules
+    restore_cache:
+      name: Restore node_modules cache
+      keys:
+        - v1-node-{{ .Branch }}-{{ checksum "package.json" }}
+        - v1-node-{{ checksum "package.json" }}
+
+  - &convert_tap_to_xunit
+    run: 
+      name: convert TAP results to xunit format
+      command:  |
+        mkdir -p tests/tape;
+        cat results.tap | npx tap-xunit > tests/tape/results.xml
+      when: always
+
+  - &save_test_metadata
+    store_test_results:
+      path: tests/tape/
+
 jobs:
   install:
     <<: *defaults
     steps:
       - checkout
       - *restore_node_modules
+      - run: npm install
```

Comment:
Isn't this redundant?

Reply:
It is indeed redundant. Squash issue, will fix. good catch!